std_msgs/Header header
float64 u_override
float64 u_safe
float64 u_nom
float64 steering_angle

float64 h2
float64 h1
float64 h0
float64 Lfh2_Lgh2usafe_ch2
float64 Lfh2_Lgh2unom_ch2
float64 Lfh1_Lgh1usafe_ch1
float64 Lfh1_Lgh1unom_ch1

float64 time_vec